rt/rs/sse/src/main/java/org/apache/cxf/jaxrs/sse/OutboundSseEventImpl.java

```diff
@@ -41,7 +41,7 @@
         private long reconnectDelay = -1;
         private Class<?> type;
         private Type genericType;
-        private MediaType mediaType;
+        private MediaType mediaType = MediaType.SERVER_SENT_EVENTS_TYPE;
```

Comment:
I would agree with @sberyozkin, `mediaType = MediaType.TEXT_PLAIN_TYPE` would be the safest option, but definitely not `MediaType.SERVER_SENT_EVENTS_TYPE`